README.md

```diff
@@ -25,15 +49,15 @@ Default environment:
 
 Required settings for relay-app:
 
-*   `INFURA_API_TOKEN`
-*   `BOND_NAME`
-*   `PACT_PRIVATE_KEY`
+*   `INFURA_API_TOKEN`: api token for connecting to Infura servers
+*   `BOND_NAME`: the bond name as seen in relay.chainweb.com
+*   `PACT_PRIVATE_KEY`: the private key for administering the bond, as described above in "Rotating the bond key" section.
 
 Usually set via default environment:
 
-*   `ETH_NETWORK_ID`
-*   `ETH_CONTRACT_ADDR`
-*   `ETH_LOCKUP_PUBLIC_KEY`
+*   `ETH_NETWORK_ID`: network identifier for mainnet or various testnets. Example: xxx
```

Comment:
Can we provide example values

Reply:
The current default is "kovan". Other possible values are "ropsten" or "mainnet". Whatever the Ethereum API supports.

Reply:
I think, that once we move to mainnet/production, the default will be "mainnet" and users won't have to worry about it. So, at this point, we can probably just delete it from the README.

Or, maybe we publish different docker images, `relay-app-testnet` and `relay-app`, so that users won't have to worry about this setting?

Reply:
```suggestion
*   `ETH_NETWORK_ID`: network identifier for mainnet or various testnets. Default is `kovan`
```